ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -201,21 +202,14 @@ open class BadgeView: UIView, TokenizedControlInternal {
         defer {
             self.dataSource = dataSource
         }
-
-        updateFonts()
```

Comment:
You might want to leave `updateFonts` here to catch any theme-wide overrides of the `BadgeView`'s font tokens

Reply:
Wait, probably not. It hasn't been really added to a theme, yet, so it can't have theme-wide overrides.